### ISSUE TITLE
Update acknowledgement

### ISSUE THREAD

**Issue by ggalibert** [aodn/imos-toolbox#657]
Currently, we have the following acknowledgement in the toolbox:
`Any users of IMOS data are required to clearly acknowledge the source of the material derived from IMOS in the format: "Data was sourced from the Integrated Marine Observing System (IMOS) - IMOS is a national collaborative research infrastructure, supported by the Australian Government." If relevant, also credit other organisations involved in collection of this particular datastream (as listed in 'credit' in the metadata record).`

And should be updated to:
`Any users of IMOS data are required to clearly acknowledge the source of the material derived from IMOS in the format: "Data was sourced from Australia's Integrated Marine Observing System (IMOS) - IMOS is enabled by the National Collaborative Research Infrastructure Strategy (NCRIS)."`

When making this change to the acknowledgement, it will be worth checking with the toolbox users, in particular AIMS as I believe they might have a slightly customised one that they would still need to update to reflect the update on IMOS datasets.

Note that the bit `If relevant, also credit other organisations involved in collection of this particular datastream (as listed in 'credit' in the metadata record).` should be removed as this was a mistake fixed in v1.4 of the IMOS NetCDF document.

**Comment by mhidas** [comment]
Has this change already been made in version 2.6.6? It's not mentioned in the [release notes](https://github.com/aodn/imos-toolbox/releases/tag/v2.6.6), so I would assume not. However, we've just received some files processed with that version, and they failed the compliance checks because they have the new acknowledgement text (and our checker plugin hasn't been updated yet).

**Comment by ocehugo** [comment]
@mhidas - 2.6.6 do not have the new string. However, this may have been modified by users for example. My branch with the modifications is not even here.

**Comment by mhidas** [comment]
@ggalibert @ocehugo I comments on the text for this attribute. Actually it's about the last sentence, after the closing double quotes, which is unchanged from before...

> If relevant, also credit other organisations involved in collection of this particular datastream (as listed in 'credit' in the metadata record).

To me, this is only an _instruction to the data provider_, and should not be included in the value of the acknowledgement global attribute. Instead, if there are other organisations to acknowledge, that should be added straight after "... with the University of Tasmania as Lead Agent."

In fact in the [IMOS NetCDF conventions](https://s3-ap-southeast-2.amazonaws.com/content.aodn.org.au/Documents/IMOS/Conventions/IMOS_NetCDF_Conventions.pdf) document (p18), this sentence is not included in the value of the attribute, only in its description.

@atkinsn I guess you might have an opinion on this too?

**Comment by ocehugo** [comment]
@mhidas, I think the content should be the actual instruction not the definition, since the practical result of this field is to instruct to any person using the netcdf file. 

TL;DR - the field should be the **instruction** on how to do proper ackn:

1. Use `Data was sourced from Australia's Integrated Marine Observing System (IMOS) - IMOS is enabled by the National Collaborative Research Infrastructure Strategy (NCRIS). It is operated by a consortium of institutions as an unincorporated joint venture, with the University of Tasmania as Lead Agent.` as a general string for ack.
2. Ack. other providers based on who published it (AIMS/CSIRO/etc).


PS: IMO, there should be a third rule there: 3. To acknowledge who is distributing it/data accessed from...

**Comment by atkinsn** [comment]
Not sure why @mhidas you say that instruction is to the data provider, it is meant for the person using the data, so that they craft the most appropriate data acknowledgement. And correct this shouldn't be included in the "acknowledgment" and hasn't been historically, as away from the metadata record it doesn't make sense.
@ocehugo why this is a fair point, we need to keep things consistent, and this is (like going back and reprocessing historical files, as we discussed a few weeks ago) something that is bigger than the acknowledgement here, and so easier just to stick with the simple update IMOS acknowledgment.

**Comment by ocehugo** [comment]
Sorry about the confusion - end of the day comment.  I was just explaining why the ack should be the way it is now (and include the questioning bit), not how they should sound.

My point was that the ack, as it is, is correct and should include the whole thing (as said, it's a direction for data users). 

I'm assuming ack. is not changed from the original msg above, so I will go forward with the PR then.

**Comment by mhidas** [comment]
> Not sure why @mhidas you say that instruction is to the data provider

Ok, I agree the overall message is targeted at the data user. However, I don't think "also credit other organisations involved in collection of this particular datastream" is a very helpful instruction for users, because that information is best known to the data _provider_ (and us).

How many users will actually go looking in the collection metadata record (which for moorings lists 4 partner organisations: CSIRO, AIMS, SARDI, SIMS), figure out which organisation produced the specific data they are using, and craft their own addition to the standard IMOS wording? Could we just provide them with a complete acknowledgement string they can copy-paste?

So, e.g. in a netCDF file from an AIMS-operated mooring, the text in the `acknowledgement` attribute could be set to something like
> "Data was sourced from Australia's Integrated Marine Observing System (IMOS) - IMOS is enabled by the National Collaborative Research Infrastructure Strategy (NCRIS). It is operated by a consortium of institutions as an unincorporated joint venture, with the University of Tasmania as Lead Agent. **Observations were obtained and quality controlled by the Australian Institute of Marine Science (AIMS).**"

Does that make sense?

**Comment by atkinsn** [comment]
@mhidas I added this statement to follow the suggested citation in the metadata record, and its from the metadata record where the user already is, so can have a look at the content in the credit fields. As most of these records are at the collection level, the statement is deliberately broad. 

I am not sure when it made its way to the NetCDF (I don't think I was involved in transferring over a metadata-specific acknowledgment to there), but yes it does take a few more steps to uncover that information. The idea of leaving it vague too is that I am sure no-one wants to go and craft very specific acknowledgment statements for each relevant data stream and keep these up to date when organisation names etc change (plus set up the checker to recognise all these different derivations). To add to the acknowledgement statement in a very deliberate fashion like this, would be something that would have to be agreed to higher up the chain (and the IMOS office). So I think this is something that @ggalibert will need to take further.

**Comment by mhidas** [comment]
Thanks @atkinsn . 

Just to be clear, we're only talking about the value of the global attribute in NetCDF files here, not about acknowledgements anywhere else.

> plus set up the checker to recognise all these different derivations

Our checker is just for the IMOS conventions, so it will only check for the standard IMOS text ("Data was sourced from Australia's Integrated Marine Observing System (IMOS) ... with the University of Tasmania as Lead Agent.") and ignore everything else.

**Comment by ggalibert** [comment]
I agree with @mhidas and I have just updated the requirements in the top description of the issue.
